controllers/categories.go

```diff
@@ -26,15 +27,19 @@ func (cc *categoryController) CreateCategory(ctx echo.Context) error {
 	var category models.Category
 	if err := ctx.Bind(&category); err != nil {
 		cc.logger.Error(err.Error())
+		log.Gl.Info("Invalid request for creating category")
```

Comment:
It's an error, not info, so use: `log.Gl.Error(err.Error)` and there is already an available `err`, no need for custom text.